elements/rh-announcement/docs/rh-announcement.md

```diff
@@ -0,0 +1,17 @@
+{% renderOverview %}
+  <rh-announcement></rh-announcement>
+{% endrenderOverview %}
+
+{% band header="Usage" %}{% endband %}
+
+{% renderSlots %}{% endrenderSlots %}
+
+{% renderAttributes %}{% endrenderAttributes %}
+
+{% renderMethods %}{% endrenderMethods %}
+
+{% renderEvents %}{% endrenderEvents %}
+
+{% renderCssCustomProperties %}{% endrenderCssCustomProperties %}
+
+{% renderCssParts %}{% endrenderCssParts %}
```

Comment:
Unrelated to this file, but you also need to add a file called `screenshot.png` to the `docs` directory with dimensions of `680x400`. This image is shown on localhost:8000 and on [All Elements](https://ux.redhat.com/elements/).

Reply:
added!